### PR TITLE
In order to make our MD files synced with having the fix by default 

### PR DESCRIPTION
**Pull request by aatwi**
1. Add a section for the live coding in the Building_Test_Data MD file 
2. Fill in the TODO sections of the Mikado_Method MD file 
3. Add new screenshots from the youtube videos 
4. Fix some text and typos in the files

### CODE REVIEW

**Review by mengdaming — APPROVED** [review]

File: docs/3_Building_Test_Data.md [hunk 1]
```diff
@@ -21,11 +21,112 @@ You'll find resource about Test Data Builders
...
+### Delete the tests
+
+In real life, you would
+
+1. Fix the bugs
+2. Use test data-builders to instantiate your test data. 
+
+So let's Ignore or Comment-Out the previous tests!
+
+### Use Mikado Method to setup data for a test on Invoice
```
Comment (philou) [inline docs/3_Building_Test_Data.md:115]: Shouldn't it be 'Test Data Builders' instead of Mikado method here?

File: docs/3_Building_Test_Data.md [hunk 2]
```diff
@@ -21,11 +21,112 @@ You'll find resource about Test Data Builders
...
+
+We'll first focus on testing the [Invoice](../java/src/main/java/com/murex/tbw/purchase/Invoice.java) class.
+It's simpler and achievable given the time we have.
+
+To solve this section, use the test entitled *Test_Data_Builders_Constraint_Applies_tax_rules_when_computing_total_amount*!
+
+Use the test data-builder to instantiate your objects in the test.
+
+Make sure your project keeps building all the time. Notice every time you have something useful.
```
Comment (philou) [inline docs/3_Building_Test_Data.md:124]: I think we should not put this ~~'Make sure your project keeps building all the time'~~ Letting them go through code that does not build highlights a shortcoming of test data builders when used alone

File: docs/3_Building_Test_Data.md [hunk 3]
```diff
@@ -21,11 +21,112 @@ You'll find resource about Test Data Builders
...
+
+### Use Mikado Method to setup data for a test on Report Generator (advanced)
```
Comment (philou) [inline docs/3_Building_Test_Data.md:126]: Again: Test Data Builders instead of Mikado Method

**Review by philou — CHANGES_REQUESTED** [review]
A few cut and paste issues

**Comment by aatwi** [comment]
@philou I updated the text as suggested

**Review by philou — APPROVED** [review]

**Pull request merged**